### PR TITLE
[4.0] mod_syndicate a11y and more

### PR DESCRIPTION
**Pull request by brianteeman**
### Problem
1. the alt text contains the word image which is not recommended
2. the alt text is hardcoded and not translatable
3. When the text is displayed then the alt text is not needed as it does not convey any meaning
4. When the text is not displayed then the use of the image as a link with an alt of feed-image does not convey the meaning of the link

### Solution
1. remove the alt text completely and replace with empty alt
2. when the text is not displayed give it a sr-only class so that the link has meaning instead of just removing it completely
3. as a result change some language strings and re-order the fields so that they make more sense and remove the show-on

### Testing
If you dont already have a syndication module then create one

The changes are visible in the config of the module
#### Before
image

#### After
image

In the frontend the changes are only visible in the markup of the module

#### After - with Label
image

#### After - without label
image

### CODE REVIEW

File: modules/mod_syndicate/tmpl/default.php [hunk 1]
```diff
@@ -14,14 +14,13 @@
 
 ?>
 <a href="<?php echo $link; ?>" class="mod-syndicate syndicate-module">
-	<?php echo HTMLHelper::_('image', 'system/livemarks.png', 'feed-image', null, true); ?>
-	<?php if ($params->get('display_text', 1)) : ?>
-		<span>
+	<?php echo HTMLHelper::_('image', 'system/livemarks.png', '', null, true); ?>
+	<?php $class = $params->get('display_text', 1) ? '' : 'sr-only'; ?>
+	<span class="<?php echo $class; ?>">
```
Comment (Quy) [inline modules/mod_syndicate/tmpl/default.php]:
```suggestion
	<?php $class = $params->get('display_text', 1) ? '' : 'class="sr-only"'; ?>
	<span <?php echo $class; ?>>
```

**Comment by particthistle** [comment]
I have tested this item :white_check_mark: successfully on 6438a9a7f695d62d54d0fa70c6d02f07dad87088<br /><br />Test successfully - results matched test instruction example results.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30188">issues.joomla.org/tracker/joomla-cms/30188</a>.</sub>

**Comment by bonzani** [comment]
I have tested this item :white_check_mark: successfully on 6438a9a7f695d62d54d0fa70c6d02f07dad87088<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30188">issues.joomla.org/tracker/joomla-cms/30188</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30188">issues.joomla.org/tracker/joomla-cms/30188</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**